app/scripts/modules/core/src/task/tasks.html

```diff
@@ -147,8 +147,9 @@ <h4>
                 </span>
               </td>
               <td>
-                {{ task.execution.authentication.user === task.getValueFor('user') ? task.execution.authentication.user
-                : (task.getValueFor('user') + ' (' + (task.execution.authentication.user || 'unknown user') + ')') }}
+                {{ task.getValueFor('user') ? ( (task.execution.authentication.user === task.getValueFor('user')) ?
```

Comment:
I know this is angular code and we're not investing in it, but I find this particularly difficult to read

Reply:
Agree. It was prettier before prettier